scriptworker/task.py

```diff
@@ -181,6 +181,40 @@ def get_revision(task, source_env_prefix):
     return revision
 
 
+def get_branch(task, source_env_prefix):
+    """Get the git branch on top of which the graph was made.
```

Comment:
Hm. Is this git only? (It looks like it, hg doesn't have `_HEAD_BRANCH`.) I'm waffling about naming this `get_git_branch` or removing the `git` reference in the docstring, since hg may have `_HEAD_BRANCH` at some point. Since I can't decide myself, this is optional.

Reply:
It can be hg too. I removed the reference to git in the docstring